src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -696,6 +739,230 @@ void M(int i)
             _ => throw null,
         };
     }
+}");
+        }
+
+        [WorkItem(37947, "https://github.com/dotnet/roslyn/issues/37947")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        public async Task TestMultiLabelWithDefault()
+        {
+            await TestInCSharp8(
+@"using System;
+
+class Program
+{
+    public static string FromDay(DayOfWeek dayOfWeek)
+    {
+        [||]switch (dayOfWeek)
+        {
+            case DayOfWeek.Monday:
+                return ""Monday"";
+            case DayOfWeek.Friday:
+            default:
+                return ""Other"";
+        }
+    }
+}",
+@"using System;
+
+class Program
+{
+    public static string FromDay(DayOfWeek dayOfWeek)
+    {
+        return dayOfWeek switch
+        {
+            DayOfWeek.Monday => ""Monday"",
+            _ => ""Other"",
+        };
+    }
+}");
+        }
+
+        [WorkItem(37949, "https://github.com/dotnet/roslyn/issues/37949")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        public async Task TestMissingOnUseInNextStatement()
+        {
+            await TestMissingAsync(
+@"using System;
+
+class Program
+{
+    public static void Throw(int index)
+    {
+        string name = """";
+        [||]switch (index)
+        {
+            case 0: name = ""1""; break;
+            case 1: name = ""2""; break;
+        }
+        throw new Exception(name);
```

Comment:
This looks like it will result in non-intuitive behavior. I'd prefer we still support this and refactor it correctly.

Reply:
This fixes the incorrect behavior by not suggesting the fix. However, for this particular case we can use the initializer value for the last arm expression as suggested in https://github.com/dotnet/roslyn/issues/37949. we can keep it open to track that case.